docs/source/api_references.rst

```diff
@@ -27,11 +27,17 @@ Core API
 .. autosummary::
     :toctree: api
     :nosignatures:
+    :template: classtemplate.rst
 
-    datamodule
-    decorators
-    hooks
-    lightning
+    ~hooks.CheckpointHooks
+    ~hooks.DataHooks
+    ~hooks.ModelHooks
```

Comment:
Are we planning to merge ModelHooks directly into the lightning module? They're not used anywhere else